Cubical/Foundations/Prelude.agda

```diff
@@ -160,6 +161,17 @@ compPathP {A = A} {x = x} {B = B} p q i =
                   (i = i1) → q j })
        (p i)
 
+
+compPathP' : ∀ {ℓ ℓ'} {A : Type ℓ} {B : A → Type ℓ'} {x y z : A} {x' : B x} {y' : B y} {z' : B z} {p : x ≡ y} {q : y ≡ z}
+             (P : PathP (λ i → B (p i)) x' y') (Q : PathP (λ i → B (q i)) y' z')
+          → PathP (λ i → B ((p ∙ q) i)) x' z'
+compPathP' {B = B} {x' = x'} {p = p} {q = q} P Q i =
+  comp (λ j → B (compPath-filler p q j i))
+       (λ j → λ { (i = i0) → x'  ;
+                   (i = i1) → Q j })
```

Comment:
indentation

Reply:
I'll try to fix this as best I can. It's right in my emacs, but I guess I can try shifting things to the left one step.

Reply:
Oh, then it's probably just the font in my browser. Let's keep it if looks good in emacs